src/main/java/com/fullcontact/api/libs/fullcontact4j/http/FCRequest.java

```diff
@@ -36,6 +36,10 @@ public String getParam(String key) {
      */
     protected abstract void makeRequest(FullContactApi api, Callback<T> callback);
 
+    protected T makeRequest2(FullContactApi api) {
```

Comment:
I left makeRequest as-is because it was used in several other classes that I didn't want to go back and change.